config/model/classifier.yaml

```diff
@@ -1,3 +1,3 @@
-# @package _group_
+_recursive_: False
```

Comment:
new option to be documented in README.md?

Reply:
This is an internal thing for hydra, not a config option that we change